src/Entity/Timeline/Measure.php

```diff
@@ -0,0 +1,238 @@
+     * @ORM\JoinTable(
+     *     name="timeline_measures_profiles",
+     *     joinColumns={
+     *         @ORM\JoinColumn(name="measure_id", referencedColumnName="id")
+     *     },
+     *     inverseJoinColumns={
+     *         @ORM\JoinColumn(name="profile_id", referencedColumnName="id")
+     *     }
+     * )
+     *
+     * @Algolia\Attribute
+     */
+    private $profiles;
+
+    public function __construct(
+        string $title,
+        string $status,
+        array $profiles = [],
+        ?string $link = null,
+        ?bool $isGlobal = false
+    ) {
+        $this->title = $title;
+        $this->status = $status;
+        $this->link = $link;
+        $this->global = $isGlobal;
+        $this->profiles = new ArrayCollection($profiles);
+    }
+
+    public function __toString()
+    {
+        return $this->title;
+    }
+
+    public function getId(): ?int
+    {
+        return $this->id;
+    }
+
+    public function getTitle(): string
+    {
+        return $this->title;
+    }
+
+    public function setTitle(?string $title): void
+    {
+        $this->title = $title;
+    }
+
+    public function getLink(): ?string
+    {
+        return $this->link;
+    }
+
+    public function setLink(?string $link): void
+    {
+        $this->link = $link;
+    }
+
+    public function getStatus(): string
+    {
+        return $this->status;
+    }
+
+    public function setStatus(?string $status): void
+    {
+        $this->status = $status;
+    }
+
+    public function getUpdatedAt(): ?\DateTime
+    {
+        return $this->updatedAt;
+    }
+
+    /**
+     * @Algolia\Attribute
+     */
+    public function getFormattedUpdatedAt(): ?string
+    {
+        if (!$this->updatedAt) {
+            return null;
+        }
+
+        return $this->updatedAt->format('Y-m-d H:i:s');
+    }
+
+    public function isGlobal(): bool
+    {
+        return $this->global;
+    }
+
+    public function setGlobal(bool $global): void
+    {
+        $this->global = $global;
+    }
+
+    public function getProfiles(): Collection
+    {
+        return $this->profiles;
+    }
+
+    public function addProfile(Profile $profile): void
+    {
+        if (!$this->profiles->contains($profile)) {
+            $this->profiles->add($profile);
+        }
+    }
+
+    public function removeProfile(Profile $profile): void
+    {
+        $this->profiles->removeElement($profile);
+    }
+
+    public function isUpcoming(): bool
+    {
+        return self::STATUS_UPCOMING === $this->status;
+    }
+
+    public function isInProgress(): bool
+    {
+        return self::STATUS_IN_PROGRESS === $this->status;
+    }
+
+    public function isDone(): bool
+    {
+        return self::STATUS_DONE === $this->status;
+    }
+
+    public function isDeferred(): bool
+    {
+        return self::STATUS_DEFERRED === $this->status;
+    }
+
+    public function equals(Measure $measure): bool
+    {
+        return $measure->getTitle() === $this->title;
```

Comment:
`$measure->getTitle()` => `$measure->title`

Reply:
IMHO, it's better to keep getTitle to not break object encapsulation

Reply:
It‘s getter for both or property for both :D